offline-virtual-machine.md

```diff
@@ -39,36 +104,35 @@ kubectl describe offlinevirtualmachine myvm
 # Look at the now created VirtualMachine status and associated events:
 kubectl describe virtualmachine myvm
 
-# Stop an OfflineVirtualMachine:
-kubectl patch offlinevirtualmachine myvm -p \
+# Stop the VirtualMachine created by this OfflineVirtualMachine:
```

Comment:
Just speak of virtual machine (avoid using the kind name): "Stop the virtual machine"

As a user you just think of it as starting and stopping a VM - you don't necessarily think about the kinds.